xc/common/utils/prjxray_create_synth_tiles.py

```diff
@@ -23,8 +23,24 @@ def map_tile_to_vpr_coord(conn, tile):
         (phy_tile_pkey, )
     )
     mapped_tiles = c.fetchall()
-    assert len(mapped_tiles) == 1, tile
-    tile_pkey = mapped_tiles[0][0]
+    assert len(mapped_tiles) > 0, tile
+    if len(mapped_tiles) > 1:
+        c.execute(
```

Comment:
This code appears to be assuming that len(mapped_tiles) in [1,2]?

Reply:
Yes, I should fix that assertion